### PR TITLE
Migrate more jobs to bootstrap json

### PR DESCRIPTION
**Pull request by fejta**
Create an execute scenario that runs a command with optional env set.

### CODE REVIEW

**Comment by rmmh** [comment]
/lgtm

We could also have a bash scenario that just executes a command directly. Not sure we need all this python boilerplate around it.

File: jobs/config.json [hunk 1]
```diff
@@ -229,6 +245,46 @@
   ]
 },
 
+"ci-test-infra-test-history": {
+  "scenario": "execute",
+  "args": [
+    "./jenkins/test_history/gen_history"
```
Comment (rmmh) [inline jobs/config.json]: this won't work. you need to cd into the directory first.

Comment (rmmh) [inline jobs/config.json]: (just make the first line of gen_history `cd "$(dirname "$0")"`)

Comment (fejta) [inline jobs/config.json]: I'm pretty sure I already did this?

Comment (rmmh) [inline jobs/config.json]: You did it in the scenario, not in gen_history.

Comment (rmmh) [inline jobs/config.json]: woops missed your gen_history changes, sorry!

Comment (fejta) [inline jobs/config.json]: The scenario doesn't cd, just gen_history?

File: scenarios/execute.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    if not cmd:
+        raise ValueError(cmd)
+    check(*cmd)
+
+
+if __name__ == '__main__':
+    PARSER = argparse.ArgumentParser()
+    PARSER.add_argument('--env', default=[], action='append')
+    PARSER.add_argument('cmd', nargs=1)
+    PARSER.add_argument('args', nargs='*')
```
Comment (krzyzacy) [inline scenarios/execute.py]: I feel like this will give people chance to append bash code to json which will be bad in the future.. Maybe do a --script?

Comment (fejta) [inline scenarios/execute.py]: I'm not super worried: it only runs a single command, it doesn't process bash variables. I think those are reasonable constraints.

**Comment by fejta** [comment]
Mostly works minute the fact that I'm not running on jenkins: https://k8s-gubernator.appspot.com/build/fejternetes/jobs/pull-cadvisor-e2e/20170114-022839-1e80eef7b01f7529-4257

**Pull request merged**